model-engine/model_engine_server/inference/post_inference_hooks.py

```diff
@@ -135,9 +135,13 @@ def handle(
             logger.warning("No firehose stream name specified. Logging hook will not be executed.")
             return
         try:
-            self._streaming_storage_gateway.put_record(stream_name=stream_name, record=data_record)
-        except StreamPutException as e:
-            logger.error(f"Error in logging hook {e}")
+            firehose_response = self._streaming_storage_gateway.put_record(
```

Comment:
nit: should we call this response something not specifically firehose-related? since we're in a file that shouldn't know about the streaming storage gateway implementation details

something like `streaming_storage_response`